contracts/Migrations.sol

```diff
@@ -1,19 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
Def agree it makes sense to delete the migrations contract / script file but just wondering if it might make sense to include an `Example.sol` (or something similarly named) and equivalent migration just for reference?

Reply:
done!